p2panda-rs/src/test_helpers.rs

```diff
@@ -0,0 +1,51 @@
+use crate::atomic::{
+    Entry, EntrySigned, Hash, LogId, Message, MessageFields, MessageValue, SeqNum,
+};
+
+/// Returns a Message for testing
+pub fn mock_message(text: String) -> Message {
```

Comment:
Maybe rename these `mock_*` methods to `create_*`? Mocking is more about "simulating" an API which actually does not do the real thing in the backend, but here we actually create "real" instances of something.

Reply:
Here is a way to use fixtures for this

https://docs.rs/rstest/0.6.4/rstest/

Reply:
nice, I'll look into this :+1: